app/controllers/admin/talentlms_controller.rb

```diff
@@ -7,28 +7,57 @@
 module Admin
   class TalentlmsController < ApplicationController
     before_action :require_can_manage_config!
+    before_action :set_config, only: [:index, :edit, :update]
 
     def index
-      @config = Talentlms::Config.first_or_initialize
+    end
+
+    def new
+      @config = config_scope.first_or_initialize
+      # If a record already exists, we want to be editing. This redirect will
+      # send the user to edit the existing record if one is found.
+      # If we ever have the need for multiple records at once, this will need to be adjusted.
+      redirect_to edit_admin_talentlm_path(@config) unless @config.new_record?
     end
 
     def create
-      @config = Talentlms::Config.first_or_initialize
-      @config.assign_attributes(config_params)
-      if config.valid?
-        @config.save
-        redirect_to action: :index
+      @config = config_scope.new
+      if @config.update(config_params)
+        respond_with(@config, location: edit_admin_talentlm_path(@config))
       else
-        render :index
+        render 'new'
       end
```

Comment:
Pretty sure `respond_with` will handle the error situation, so this can probably just be:
```
@config = config_scope.create(config_params)
respond_with(@config, location: edit_admin_talentlm_path(@config))
```